src/posts/2021-09-03-getting-started-with-orca.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Getting started with Orca screen reader on Gnome desktop environment on Ubuntu 20.04 LTS
+description: A quick overview of getting started with Orca on Gnome desktop environment on Ubuntu 20.04 LTS.
+category: Assistive technology
+author: Joseph mawa
+date: 2021-09-03
+further_reading:
+  - title: Orca screen reader documentation
+    url: https://help.gnome.org/users/orca/stable/index.html.en
+    source: Gnome foundation
+  - title: Orca screen reader tutorial
+    url: https://techblog.wikimedia.org/2020/07/02/an-orca-screen-reader-tutorial/
+    source: Wikimedia foundation
+tags:
+  - assistivetechnology
+---
+
+## Introduction to Orca
+
+Orca is a free, open-source screen reader developed under the GNOME project. Several Linux distributions have it installed by default.
+
+This article will introduce you to the basics of Orca on Gnome desktop environment on Ubuntu version 20.04 LTS. There might be some slight variations if you are using a different desktop environment or using GNOME on a different Linux distribution. You need to consult the manual for your Linux distribution or desktop environment.
+
+## Launching Orca
+
+There are three ways you can launch Orca on Ubuntu 20.04 LTS. Below are the descriptions of these three methods.
+
+- Use the keyboard shortcut <kbd>Super</kbd> + <kbd>Alt</kbd> + <kbd>S</kbd>.
+- Type `orca` on a terminal window and then press the <kbd>Return</kbd> key. Alternatively, you can type `orca` in the Run dialog and press the <kbd>Return</kbd> key. You can use the key combination <kbd>Alt</kbd> + <kbd>F2</kbd> to open the Run dialog and the <kbd>ESC</kbd> key to close it. If you launch Orca using the Run dialog, you can close it by typing the command `pkill orca` or `killall orca` on the terminal or in the the Run dialog and then pressing the <kbd>Return</kbd> key.
+- Open the Settings dialog and then select Universal access. You can then toggle the screen reader on or off.
+
+After launching Orca, it will immediately start narrating the content of the focused application.
```

Comment:
Can we rephrase "focused application"? The reason I'm suggesting this is it may be a concept a reader may be unfamiliar with.

Reply:
Thanks for pointing this out once again. Orca actually announces the title of the topmost open application window. And that is only if the app is accessible to Orca( Most Gnome apps are). I am afraid there are quite a number of third-party linux and cross-platform apps that are not accesible to Orca. I have made that a bit clearer now.